src/main/java/it/gov/pagopa/payhub/activities/service/classifications/TransferClassificationStoreService.java

```diff
@@ -138,6 +138,14 @@ public Integer saveClassifications(
 				.transferCategory(optionalTransfer.map(Transfer::getCategory).orElse(null))
 				.receiptCreationDate(optionalReceipt.map(ReceiptNoPII::getCreationDate).orElse(null))
 				.installmentBalance(optionalInstallment.map(InstallmentNoPII::getBalance).orElse(null))
+				.pspLastName(optionalTreasury.map(Treasury::getPspLastName).orElse(null))
```

Comment:
pspLastName is already set on line 118

Reply:
ps: can we order all the set field on this method in order to group by source, separating each group by an empty line?